docs/RELEASE_NOTES.md

```diff
@@ -35,6 +35,18 @@
     - **Product Sync**: `ProductDraft.getState()` is changed from `Reference<State>` to `ResourceIdentifier<State>`, so as a library user you don't need to provide a key field in the id field of the Reference. (Now API and JVM SDK support `ResourceIdentifiers` and it supports id or key as a field). [#589](https://github.com/commercetools/commercetools-sync-java/pull/589)
 
 -->
+### 2.4.0 - Nov 05, 2020
```

Comment:
Can you check the version above mentioned with 3.0.0?

Reply:
I think it is just there as a template.

Reply:
No, it's not a template but commented because not released yet, and the code is on the master.

Reply:
Oh okay, I think now I got it. So in this release we are going to release shopping list plus the breaking changes which are commented here (and already in the master branch), meaning that the new release would be 3.0.0?

Reply:
This is correct. New release will be v3.0.0.

Reply:
I changed it here: 859a9640